### ISSUE TITLE
Personalization: The default dark color was changed to gray

### ISSUE THREAD

**Issue by anhallbe** [infor-design/enterprise#6512]
**Describe the bug**
Not sure if this is by design or if it's a bug, but the default dark color appears to have been changed from blue (azure) to gray (slate). This includes the color of headers, buttons and possibly other elements

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/button/example-index.html?theme=new&mode=dark
2. Notice that all the buttons are gray
3. Hover a primary button
4. The hovered primary button is blue

**Expected behavior**
The change of the default color has a pretty big impact on the look of an app. May be considered a breaking change even? Either way, it looks a bit off that some elements are now gray by default, and gets another color when hovered.

**Version**

- ids-enterprise: 4.63.2

**Screenshots**
[4.63.2](https://main-enterprise.demo.design.infor.com/components/button/example-index.html?theme=new&mode=dark):
<img width="853" alt="image" src="https://user-images.githubusercontent.com/4991954/171184678-186aa12f-8569-42c8-97da-2e0d9246e125.png">

[4.63.0](https://design.infor.com/code/ids-enterprise/4.63.0/demo/components/button/example-index?theme=new&mode=dark):
<img width="853" alt="image" src="https://user-images.githubusercontent.com/4991954/171185231-984a7ce0-5d42-4c23-ac1b-ff684953871b.png">

**Comment by tmcconechy** [comment]
I suspect that the buttons are a bug. But need to check. cc @inforandy

**Comment by tmcconechy** [comment]
To fix this just make the default for dark mode the same as https://main-enterprise.demo.design.infor.com/components/button/example-index.html?colors=606066&theme=new&mode=dark

And after checking with design @anhallbe it was the intention to make the buttons the same color as the color theme (header color) but we never had it 100^ correct

**Comment by janahintal** [comment]
QA Passed. Verified in https://main-enterprise.demo.design.infor.com/components/button/example-index.html?theme=classic&mode=dark&colors=2578a9
![image](https://user-images.githubusercontent.com/45757016/183595913-92e1adba-f1bd-4e61-9c9c-46a072386eb4.png)
![image](https://user-images.githubusercontent.com/45757016/183596049-beed6650-c495-4d19-abac-61d67de1fa7a.png)

**Comment by danielortiz1982** [comment]
As per Andy reopening till we figure this out. 
CC: @inforandy @tmcconechy

**Comment by inforandy** [comment]
Sorry for the confusion here; was looking at this one earlier today alongside the issue the portal team sent you an email about @tmcconechy, regarding dark mode tag colorations being grayscale instead of blue. 

I suspect we have some misalignment somewhere on what the expected dark mode color transformations are, so I flagged this one to recheck as well when I recalled we had been discussing it. 

We can keep this closed for the time being, since it's done, and then just dive back into dark mode overall to make sure everything is behaving as expected.
